x/auth/client/cli/tx_sign.go

```diff
@@ -233,20 +237,19 @@ func makeSignCmd() func(cmd *cobra.Command, args []string) error {
 		if err != nil {
 			return err
 		}
-		from, _ := cmd.Flags().GetString(flags.FlagFrom)
-		_, fromName, _, err := client.GetFromFields(txF.Keybase(), from, clientCtx.GenerateOnly)
-		if err != nil {
-			return fmt.Errorf("error getting account from keybase: %w", err)
-		}
 
 		overwrite, _ := f.GetBool(flagOverwrite)
 		if multisig != "" {
 			multisigAddr, _, _, err := client.GetFromFields(txFactory.Keybase(), multisig, clientCtx.GenerateOnly)
```

Comment:
Here, the 2nd argument is the multisig name, and we should pass this 2nd argument into `SignTxWithSignerAddress`. We dont need the `multisigKey, err := txFactory.Keybase().KeyByAddress(multisigAddr)` line.